src/aws-redshiftserverless/workgroup.ts

```diff
@@ -0,0 +1,351 @@
+import { IResource, Lazy, Names, Resource, aws_redshiftserverless, aws_ec2, Token, Stack } from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import { INamespace } from './namespace';
+
+/**
+ * A Redshift Serverless Workgroup
+ */
+export interface IWorkgroup extends IResource, aws_ec2.IConnectable {
+  /**
+   * The workgroup Arn
+   *
+   * @attribute
+   */
+  readonly workgroupArn: string;
+  /**
+   * The workgroup name
+   *
+   * @attribute
+   */
+  readonly workgroupName: string;
+  /**
+   * The workgroup id
+   *
+   * @attribute
+   */
+  readonly workgroupId: string;
+  /**
+   * The workgroup endpoint address
+   *
+   * @attribute
+   */
+  readonly endpointAddress: string;
+  /**
+   * The workgroup port
+   *
+   * @attribute
+   */
+  readonly port: number;
+}
+
+/**
+ * Properties for defining a Redshift Serverless Workgroup.
+ */
+export interface WorkgroupProps {
+  /**
+   * The base compute capacity of the workgroup in Redshift Processing Units (RPUs).
+   *
+   * You can adjust the Base capacity setting from 8 RPUs to 512 RPUs in units of 8.
+   * Also you can increment or decrement RPUs in units of 32 when setting a base capacity between 512-1024.
```

Comment:
Ref: [Amazon Redshift Serverless now supports higher base capacity of 1024 Redshift Processing Units
](https://aws.amazon.com/about-aws/whats-new/2024/09/amazon-redshift-serverless-capacity-1024-processing-units/)